Makefile

```diff
@@ -133,12 +133,10 @@ docker: _print_vars
 	docker build -t $(IMG) -f ./build/Dockerfile .
 	docker push $(IMG)
 
-### webhook: generate certificates for webhooks and deploy to cluster; no-op if webhooks are disabled or running on OpenShift
+### webhook: generate certificates for webhooks and deploy to cluster; no-op if running on OpenShift
 webhook:
-ifeq ($(WEBHOOK_ENABLED),true)
 ifeq ($(TOOL),kubectl)
 	./deploy/webhook-server-certs/deploy-webhook-server-certs.sh kubectl
```

Comment:
I wonder if it makes sense optionally generate TLS certs as previously but instead extract volume and volumes bound from main k8s controller yaml, like:
`kubectl patch -f ./deploy/k8s/controller-tls.yaml --merge` ->
```
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: che-workspace-controller
  namespace: che-workspace-controller
spec:
  template:
    spec:
      containers:
        - name: che-workspace-controller
          volumeMounts:
            - name: webhook-tls-certs
              mountPath: /tmp/k8s-webhook-server/serving-certs
              readOnly: true
      volumes:
        - name: webhook-tls-certs
          secret:
            secretName: webhook-server-tls
```
Not sure if it's going to work

Reply:
I was able to get this to work and have updated the PR!